queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM vet_clinic WHERE name LIKE '%mon%';
+SELECT * FROM vet_clinic WHERE date_of_birth BETWEEN '2016-1-1' AND '2019-1-1';
+SELECT * FROM vet_clinic WHERE escape_attempts < 3;
+SELECT date_of_birth FROM vet_clinic WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM vet_clinic WHERE weight_kg > 10.5;
+SELECT * FROM vet_clinic WHERE neutered = TRUE;
+SELECT * FROM vet_clinic WHERE name != 'Gabumon';
+SELECT * FROM vet_clinic WHERE weight_kg <= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] This query suppose to find all animals with a weight between 10.4kg and 17.3kg, So please correct the condition where `weight_kg <= 10.4` to be `weight_kg >= 10.4` to fulfill this query.

Reply:
It says in the requirements that it should be (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

![Screenshot 2023-01-16 at 11 53 25 PM](https://user-images.githubusercontent.com/63551733/212771100-43aeab98-d115-4072-a7d7-54e5ccc72ad8.png)